youtube_dl/extractor/bellesa.py

```diff
@@ -0,0 +1,109 @@
+            'creator': 'Bellesa Films',
+            'upload_date': '20191216',
+            'timestamp': 1576539207,
+            'duration': 721,
+            'tags': ['HD Porn', 'Porn for Women', 'Orgasm', 'Bellesa Films', 'Threesome', 'FFF', 'Girl on Girl', 'Lesbians', 'Lesbian Porn', 'Nipple Licking', 'Finger', 'Cunnilingus', 'Anilingus', 'Eating Out', 'Clit Play', 'Clit Stimulation', 'Natural Breasts', 'Face Sitting', 'Spitting'],
+            'categories': ['Girl on Girl', 'Story'],
+            'age_limit': 18,
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(
+            'https://www.bellesa.co/videos/%s/' % video_id, video_id)
+
+        # videos on this page are embedded into a container called VideoCard - if there is
+        # nothing on the page referencing a VideoCard we cannot extract the information and
+        # thus need to raise an error
+        if 'VideoCard' not in webpage:
+            title = self._html_search_regex(
+                r'<title[^>]*>(?P<title>.+?)\s+\|\s+Bellesa',
+                webpage, 'title', default=None,
+                group='title', fatal=False)
+
+            raise ExtractorError('%s said: %s' % (self.IE_NAME, clean_html(title)), expected=True)
+
+        initial_data_raw = self._search_regex(r'window\.__INITIAL_DATA__\s+=\s+(.+?);</script>', webpage, 'initial_data')
+
+        try:
+            initial_data = json.loads(initial_data_raw)
+        except json.JSONDecodeError:
+            raise ExtractorError('%s said: cannot decode initial data', self.IE_NAME, expected=True)
+
+        video = try_get(initial_data, lambda x: x['video'])
+        if not video:
+            raise ExtractorError('%s said: initial data malformed' % self.IE_NAME, expected=True)
+
+        resolutions = try_get(video, lambda x: x['resolutions'])
+        source = try_get(video, lambda x: x['source'])
+
+        if not resolutions or not source:
+            raise ExtractorError('%s said: cannot extract playlist information from meta data' % self.IE_NAME, expected=True)
+
+        m3u8_url = 'https://s.bellesa.co/hls/v/%s/,%s,.mp4.urlset/master.m3u8' % (source, resolutions)
+        formats = self._extract_m3u8_formats(
+            m3u8_url, video_id, 'mp4',
+            entry_protocol='m3u8_native', m3u8_id='hls',
+            fatal=False)
+
+        self._sort_formats(formats)
+
+        # get from video meta data first
+        title = video.get('title')
+        if title:
+            title = title.strip()
```

Comment:
Condition the value:
```suggestion
        title = strip_or_none(video.get('title'))
        if title:
```